.gitignore

```diff
@@ -9,3 +9,4 @@ _scratch/
 /spec/gen_spec/spec*.db
 tags
 /tests/debug.py
+.idea/
```

Comment:
good point - makes sense. This needs to be configured, as git does not make it by default.